app/nexus/internal/state/base/policy.go

```diff
@@ -268,61 +307,22 @@ func ListPoliciesByPath(pathPattern string) []data.Policy {
 //     an empty slice if no policies match. The order of policies in the
 //     returned slice is non-deterministic due to the concurrent nature of the
 //     underlying store.
-func ListPoliciesBySPIFFEID(SPIFFEIDPattern string) []data.Policy {
-	var result []data.Policy
-
-	policies.Range(func(key, value interface{}) bool {
-		policy := value.(data.Policy)
-		if policy.SPIFFEIDPattern == SPIFFEIDPattern {
-			result = append(result, policy)
-		}
-		return true
-	})
-
-	return result
-}
-
-// ImportPolicies imports a set of policies into the application's memory state.
-// It validates each policy, ensuring it has compiled regex patterns before
-// storing.
-//
-// Parameters:
-//   - importedPolicies: A map of policy IDs to policy objects
-//
-// Returns:
-//   - error: An error if any policy fails validation
-func ImportPolicies(importedPolicies map[string]*data.Policy) {
-	for id, policy := range importedPolicies {
-		// Skip nil policies.
-		if policy == nil {
-			continue
-		}
```

Comment:
we don't import policies anymore since we fetch them fresh from the db whenever we need it. -- import was to import them into the memory.